hibernate-core/src/main/java/org/hibernate/dialect/AbstractHANADialect.java

```diff
@@ -1497,8 +1501,11 @@ public void contributeTypes(TypeContributions typeContributions, ServiceRegistry
 					}
 				}
 			}
-			catch (SQLException e) {
-				// ignore
+			catch (Exception e) {
+				LOG.debug(
+						"An error occurred while trying to determing the value of the HANA parameter indexserver.ini / session / max_lob_prefetch_size. Using the default value "
```

Comment:
@vladmihalcea just noticed a typo here: s/determing/determine/

Reply:
Yeah, don't bother, I'd better push a fix myself :). Too used to the NoORM process :]

Reply:
Just pushed the fix.

Reply:
ok :-)